kitsune/questions/static/questions/js/components/AAQStep.jsx

```diff
@@ -28,15 +31,21 @@ export default class AAQStep extends React.Component {
     return null;
   }
 
-  scrollToNextStep() {
+  visible(){
+    return false;
+  }
+
+  switchToNextStep() {
     let currentStep = React.findDOMNode(this); // will throw if not mounted
     let nextStep = currentStep.nextElementSibling;
     if (nextStep === null) {
-      throw 'Tried to call scrollToNextStep(), but this is the last step.';
+      throw 'Tried to call switchToNextStep(), but this is the last step.';
     }
     if (!nextStep.classList.contains('AAQApp__Step')) {
-      throw 'Tried to call scrollToNextStep(), but the next element is not an AAQStep.';
+      throw 'Tried to call switchToNextStep(), but the next element is not an AAQStep.';
     }
+    nextStep.style.display = 'block';
+    currentStep.style.display = 'none';
```

Comment:
This kind of manipulation generally isn't safe in React. If the components re-render, this will easily be erased. Anything that affects the DOM like this should be in a `render()` method.

I think a better approach would be to have a state variable that lists the current active step, and render just that one instead of manipulating the visibility of each step.